src/import/FSHImporter.ts

```diff
@@ -1933,26 +1933,36 @@ export class FSHImporter extends FSHVisitor {
       case VsOperator.EQUALS:
       case VsOperator.IN:
       case VsOperator.NOT_IN:
-        if (typeof value !== 'string') {
-          throw new ValueSetFilterValueTypeError(operator, 'string');
+        // NOTE: We believe that =, in, and not-in operators should ONLY support code values
+        // based on the filter value documentation:
+        // http://hl7.org/fhir/R4/valueset-definitions.html#ValueSet.compose.include.filter.value
+        // Both string and code are supported for now in order to maintain backwards compatibility.
+        if (typeof value === 'string') {
+          logger.warn(
+            `The match value of filter operator "${operator}" must be a code. ` +
+              'Support for string values has been deprecated and will be removed in a future release.'
+          );
+        }
```

Comment:
So I've gone back and forth on this a lot, but it feels a little bit harsh to me for us to go from _requiring_ `string`, to now saying that `string` is deprecated, and you have to use `code`. Especially since there isn't really a super strong reason why we can't allow `string`, the `code` just gets turned into a `string` anyway. I know there is an argument for readability of having the types be explicit in FSH, but in this case I'm not sure the upside is worth deprecating something that we really could just leave in. But I could go either way honestly, and I've already changed my mind multiple times. What do others thing?

Reply:
Also not just deprecating something, but deprecating something that we know people must have been using, because it was the only way to use an `in`, `=` or `not-in` operator.

Reply:
This is a fair point. I think based on my reading of the definition (which admittedly I'm not very confident in), the type we _should_ be supporting is just `code`. In that case, we should somehow start advising people to shift from `string` to `code`. Maybe deprecating is too harsh and we should just say it is not advised to use a string and update to a code?

Reply:
I am... also unsure of this. As @ngfreiter said, the `code` will only be turned into a `string` anyways so I lean on the side of not deprecating this. I think a simple warning letting users know that the `string` will be interpreted as a `code`, but that using a `string` value for these operators is inappropriate would suffice.

Reply:
Based on our conversation this morning, I've kept the warning message for deprecating `string` values and updated the warning message. I also reverted the support for informing users about a choice of values since we only want them using a `code` for the majority of the filter operators. Let me know what you think.